curated_transformers/models/albert/_hf.py

```diff
@@ -74,29 +75,42 @@ def convert_hf_state_dict(params: Mapping[str, Parameter]) -> Mapping[str, Tenso
         out[name] = parameter
 
     # Rename and move embedding parameters to the inner BertEmbeddings module.
-    out["embeddings.word_embeddings.weight"] = stripped_params[
-        "embeddings.word_embeddings.weight"
-    ]
-    out["embeddings.token_type_embeddings.weight"] = stripped_params[
-        "embeddings.token_type_embeddings.weight"
-    ]
-    out["embeddings.position_embeddings.weight"] = stripped_params[
-        "embeddings.position_embeddings.weight"
-    ]
-    out["embeddings.layer_norm.weight"] = stripped_params["embeddings.LayerNorm.weight"]
-    out["embeddings.layer_norm.bias"] = stripped_params["embeddings.LayerNorm.bias"]
-
-    # Embedding projection
-    out["embeddings.projection.weight"] = stripped_params[
-        "encoder.embedding_hidden_mapping_in.weight"
-    ]
-    out["embeddings.projection.bias"] = stripped_params[
-        "encoder.embedding_hidden_mapping_in.bias"
-    ]
+    key_map = {
```

Comment:
Now that these are nicely bundled, it's probably better so make a top-level `KEY_MAP` constant. We could probably use

https://docs.python.org/3/library/types.html#types.MappingProxyType

if we are worried about external modification?